tools/runtime-cert/checks/pacs_check.py

```
@@ -0,0 +1,205 @@
+        errors.append(f"Contract version mismatch: expected '{EXPECTED_CONTRACT_VERSION}', got '{contract.get('version')}'")
+
+    # Validate database connectivity
+    databases = proof.get("databases", {})
+    if databases.get("pacs_oltp") != "reachable":
+        errors.append(f"pacs_oltp not reachable: {databases.get('pacs_oltp')}")
+
+    # Validate views (no "missing" status allowed)
+    views = proof.get("views", {})
+    for view in REQUIRED_VIEWS:
+        # Convert view name to response key format
+        key = view.replace("vw_TerraFusion_", "vw_terra_fusion_").replace("_", "")
```

Flake8 (7.3.0)

[error] local variable 'key' is assigned to but never used

(F841)